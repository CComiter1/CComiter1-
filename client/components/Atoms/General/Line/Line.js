import styled from 'styled-components'

export default styled.hr`
    width: 90%;
    height: 4px;
    background-color: ${prop => prop.theme.secondaryColor};
    `
