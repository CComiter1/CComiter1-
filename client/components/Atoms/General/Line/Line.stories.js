import React from 'react'
import { storiesOf } from '@storybook/react'

import Line from './Line'

storiesOf('Atoms/General', module)
  .add('Line', () => (
    <Line />
  ))
