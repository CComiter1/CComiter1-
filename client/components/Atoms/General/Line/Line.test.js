import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import Line from './Line'

describe('Line', () => {
  it('renders correctly', () => {
    const component = renderer.create(
      <Line />
    ).toJSON()
    expect(component).toMatchSnapshot()
  })
})
