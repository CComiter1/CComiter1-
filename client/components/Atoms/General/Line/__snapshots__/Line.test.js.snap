// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Line renders correctly 1`] = `
.c0 {
  width: 90%;
  height: 4px;
}

<hr
  className="c0"
/>
`;
